### DISCUSSION TITLE
Vercel not accpting payment kindly help with workarround

### DISCUSSION THREAD

**Discussion by myperfectfit-dev** [vercel/community#4507] [Account Issue] [answered]
### Summary

Hi ,

The vercel system despite giving correct card details throwing error and not accepting and not allowing to clear an invoice what is the best way to get the payment done. reaching out to support still no use as they are not respoding, reached out to stripe still no use. Please help me to get the payment done any one has support phone number to reach them.

Thanks,
Raj

### Steps to Reproduce

By clicking on the charge now its showing

 An unexpected internal error occurred

**Comment by myperfectfit-dev** [comment]
00165632 is my ticket number

**Reply by amyegan** [reply]
Sorry you're having trouble with your payment method. It looks like the support team responded on your ticket recently.

**Reply by myperfectfit-dev** [reply]
Actually suport said to check with some email id .. but the thing is new card is also not accpting . I am tryign with new card the site is not working for that also contineously its throwing error. Can you please help me

**Reply by myperfectfit-dev** [reply]
Hi ,

I have got the email but the issue is still their. We need to add new card and then do the payment. Currently the email which you generated is not showing to pay any thing. Kindly give us a new link where i can add a card their by i can pay thigns. Support needs to generate a new link with the new card payment. kindly help us please our site is down and we are loosing customers.

**Reply by myperfectfit-dev** [reply]
This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.
this is the error i am getting now

**Reply by myperfectfit-dev** [reply]
We are having very hard time in paying the invoice payment to vercel..

New card addition failing
Old card payment failing
Support escalation not workign
Our systems are down
No one from sales team respoinding to upgrade the account to enterprise
Support team in the discusion forum sending us the same email link to pay which is failing agian again again agianagin
2 days we are having issue.

Example
No response

Steps to Reproduce
Go to Payment page add newcard showing error
This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.

Pay the invoice with existing card showing error
This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.

**Reply by amyegan** [reply]
I'm sorry you're still having trouble. Please follow up with ticket 00166188. The team handling your case can access the billing system to get this issue solved 🙇

**Reply by myperfectfit-dev** [reply]
Hi,

Last time when it happened dan from support gave us a new link to process the payment. The issue is support saying contact bank ... we have contacted bank they are allowing things and i have internaltion card which i have paid to vercel previously. Kindly genereate a link and give us. Our systems are down badly from 2 days business is impacted. Kindly kindly kindly kindly kindly kindly help us to the payment made to vercel. Support are not getting what i am asking for.

![image](https://github.com/vercel/community/assets/75773373/da326d37-6082-4ca7-81a1-68d69e461b09)

**Reply by myperfectfit-dev** [reply]
Whats happening now  i cant pay as system is not all seems to check the card.. the same card is used to pay using the other account, Kindly generate new link for the payment and send us. This is a small issue from your end. Please address fast as its been 2 days we are still waiting

![image](https://github.com/vercel/community/assets/75773373/01a4b72d-4ade-43c6-bac6-18cdf26aead3)

**Reply by myperfectfit-dev** [reply]
Support team are not responding to our requests. @amyegan kindly help us @swarnava can you please raise a new link for payment please

**Reply by myperfectfit-dev** [reply]
Kindly help us to resolve this error or payment issue as soon possible

**Comment by myperfectfit-dev** [comment]
I am trying to reach you to resolve our issue of the payment to vercel
#00166188 <https://vercel.com/teams/mpf-dev/support/cases/00166188>

I am getting the following error

This invoice can no longer be paid. Consider voiding, marking as
uncollectible, or marking as paid out of band instead.


The email which you sent to accounts team is also showing error.



Can you please send me a link where i can enter own card details and pay


Thanks,

Rajesh KUmar.

**Comment by swarnava** [comment, marked as answer]
Hi, We are responding to you via email and closing this discussion.

**Comment by myperfectfit-dev** [comment]
The issue is still not resolved as the payment is stuck in the payment page

[image: image.png]

Can you please guide

**Comment by myperfectfit-dev** [comment]
Issue is resolved now finally.
